db/dbConnection.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose')
+
+function dbConnection(){
+mongoose.connect('mongodb+srv://andleebsyed:andydev%40123@neog-cluster.4e6nl.mongodb.net/inventory' ,  {
```

Comment:
Move the URI to environment variables, so that you don't expose sensitive data like your password.